src/test-utils/tsconfig.json

```diff
@@ -11,5 +11,5 @@
     "experimentalDecorators": true,
     "tsBuildInfoFile": "../../.cache/test-utils.tsbuildinfo"
   },
-  "include": ["./", "../../types"]
+  "include": ["./", "./types"]
```

Comment:
Created separate `*.selectors.js` type declarations for test-utils folder, because there is a difference between `"module": "CommonJS"` (used for compilation) and `"module": "ESNext",`  (used in tests)